tests/test_integration.py

```diff
@@ -111,7 +111,7 @@ def test_same_bqm(self):
         """Run demo.py and demo_numpy.py with same inputs to check same BQM created."""
 
         w, h = (random.randint(10,20), random.randint(10,20))
-        num_poi, num_cs, num_new_cs = (random.randint(1,4), random.randint(1,4), random.randint(1,4))
+        num_poi, num_cs, num_new_cs = (random.randint(2,4), random.randint(2,4), random.randint(2,4))
```

Comment:
If we do end up needing to enforce that `num_new_cs>1`, it should be explained in the docstring why.

And then if that's the case, in this test, I think we'd only need to do:
```suggestion
        num_poi, num_cs, num_new_cs = (random.randint(2,4), random.randint(2,4), random.randint(1,4))
```